enhancements/storage/csi-driver-operator-merge.md

```diff
@@ -514,6 +514,8 @@ azure-disk-csi-driver-operator=<the scratch build> \
 oc adm release extract --command openshift-install quay.io/jsafrane/scratch:release1
 ```
 
+This step is only applicable for CVO based operators and not OLM based operators. For OLM based operator - either an image can be built locally and deployed using your personal index image or you can ask ART team for a scratch image when you open `ocp-build-data` PR and proceed to include that image in your personal index image.
```

Comment:
This seems a little too condensed. Can you actually include the operator image in index? I think you can include only bundles.

What we actually do is that we specify the operator image in ClusterServiceVersion, make a bundle out of it, and include the bundle in the index image. For that we have a script, so the scratch image would be used like this:

```
./create-efs-bundle registry.ci.openshift.org/ocp/4.9:aws-efs-csi-driver <OPERATOR_SCRATCH_IMAGE> quay.io/<my-repo>/efs-bundle quay.io/<my-repo>/efs-index
```